folly/Format-inl.h

```diff
@@ -471,7 +471,7 @@ class FormatValue<
 
       valBufBegin = valBuf + 3;  // room for sign and base prefix
       int len = snprintf(valBufBegin, (valBuf + valBufSize) - valBufBegin,
-                         "%'ju", static_cast<uintmax_t>(uval));
```

Comment:
' provides thousands separation.  So I would say the very least would be something along the lines of:

```
#ifdef FOLLY_NO_PRINTF_GROUPING
  "%ju",
#else
  "%'ju",
#endif
  static_cast<uintmax_t>(uval));
```

And add a configure check to conditionally set that define.

Better would be to implement a thousands separating output for non-gnu platforms, but that's a bit more work.